tests/EventListener/AddFormatListenerTest.php

```diff
@@ -32,7 +32,7 @@ public function testNoResourceClass()
         $eventProphecy->getRequest()->willReturn($request)->shouldBeCalled();
         $event = $eventProphecy->reveal();
 
-        $listener = new AddFormatListener(new Negotiator(), ['notexist' => 'application/vnd.notexist']);
+        $listener = new AddFormatListener(new Negotiator(), ['notexist' => 'application/vnd.notexist'], []);
```

Comment:
As the new array argument has a default value we don't really need to change this right?